pkgs/development/libraries/libunwind/default.nix

```diff
@@ -2,21 +2,13 @@
 
 stdenv.mkDerivation rec {
   pname = "libunwind";
-  version = "1.6.2";
+  version = "1.7.2";
 
   src = fetchurl {
-    url = "mirror://savannah/libunwind/${pname}-${version}.tar.gz";
-    sha256 = "sha256-SmrsZmmR+0XQiJxErt6K1usQgHHDVU/N/2cfnJR5SXY=";
+    url = "https://github.com/libunwind/libunwind/releases/download/v${version}/libunwind-${version}.tar.gz";
```

Comment:
Should we use their provided tarballs, or should we rather use `fetchFromGitHub`? I can't seem to find any CI job creating those tarballs.

Reply:
Updated